### ISSUE TITLE
Update Stripe PHP bindings

### ISSUE THREAD

**Issue by laryn** [backdrop-contrib/stripe#24]
We currently include `v7.29.0` which is getting pretty old.

**Comment by quicksketch** [comment]
I need a newer version yet, as I want to use the Customer Portal, which doesn't seem to be present in the 7.x version of the Stripe API: https://stripe.com/docs/api/customer_portal

**Comment by laryn** [comment]
@jenlampton @quicksketch I'm not using this module anywhere currently but am considering it, and have submitted a PR that will update the Stripe PHP bindings to the latest (as of a few days ago). The module's Bitcoin and Alipay settings have also been removed since the respective Stripe settings were deprecated and removed in the interim. I've tested this PR at a most basic level on a webform by creating a test form with a test payment option and submitting it with a test credit card in test mode (you get the idea). That seems to be working fine.

If you're using this module on a site anywhere, are you able to give this PR a trial run?
